tests/web-demo/for.egg

```diff
@@ -1,23 +0,0 @@
-(relation range (i64))
```

Comment:
We deleted this resugaring of it broke on a new egglog version, unrelated to our changes. cc @oflatt @yihozhang FYI we should deal with this at some point. The forbidding of `@` in user defined code broke this test.